manifests/params.pp

```diff
@@ -441,7 +441,7 @@
   $server_idle_timeout                    = 1200000
   $server_web_idle_timeout                = 30000
   $server_connect_timeout                 = 120000
-  $server_enable_ruby_profiler            = false
+  $server_enable_ruby_profiler            = true
```

Comment:
I can only cite the docs: `Puppet Server enables profiling for the Puppet Ruby code.` For < 5.0 this was disabled by default, but for 5.x it's enabled by default because the metrics that are coming newly with 5.x seem to depend on it, at least partly.

I can only guess that probably nobody really used this option. ;)

When further applying this logic: on a 2.7 server, the metrics switch would then only enable Ruby profiling, as there's no metrics service... via CHANGELOG entry and a migration in foreman-installer, people having the profiler enabled could get the metrics parameter enabled.